.travis.yml

```diff
@@ -6,17 +6,31 @@ python:
 # Use new Travis infrastructure (Docker can't sudo yet)
 sudo: false
 
-# No need to install any deps.
-install: true
+# Install coveralls to obtain code coverage
+install:
+  - "pip install coveralls"
+  - "pip install pep8"
```

Comment:
This line can/should come out. The install of flake8 will pull in pep8 as a dependency. Also, this will help as I am getting ready to rename pep8 -> pycodestyle.